app/assets/javascripts/js-routes.js.erb

```diff
@@ -1,2 +1,3 @@
 <%# encoding: UTF-8 %>
+<%= depend_on "file://#{Rails.root.join 'config', 'routes.rb'}" %>
```

Comment:
Nothing, from my tests. I've based on this test of sprockets:

https://github.com/rails/sprockets/commit/763d2a956d61705b63a0aba3608392d38bdafeb1

This test was added to sprockets since 3.0.0.